raiden/api/rest.py

```diff
@@ -258,12 +259,13 @@ def open(self, partner_address, token_address, settle_timeout, balance=None):
                 )
             except EthNodeCommunicationError as e:
                 return make_response(str(e), httplib.REQUEST_TIMEOUT)
+            except InsufficientFunds as e:
+                return make_response(str(e), httplib.CONFLICT)
```

Comment:
We could return even [`402 Payment Required`](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors) whenever user has insufficient token or ETH for gas funds.

I am just a bit hesitant about it since the page says "Reserved for future use". 
@hackaugusto opinion?